pl_bolts/metrics/aggregation.py

```diff
@@ -29,5 +29,5 @@ def precision_at_k(output, target, top_k=(1,)):
         res = []
         for k in top_k:
             correct_k = correct[:k].contiguous().view(-1).float().sum(0, keepdim=True)
-            res.append(correct_k.mul_(100.0 / batch_size))
+            res.append(correct_k.mul_(100.0).mean())
```

Comment:
ok, this seems to solve the bug, but not sure if the problem is somewhere else...
cc: @ananyahjha93 @akihironitta